ApiClient/Source/Pages/Queries/SearchQuery.swift

```diff
@@ -44,13 +44,13 @@ final class SearchQuery: Query {
     }
 
     /// Initiliaze with a query string
-    convenience init(query: String) {
+    public convenience init(query: String) {
         self.init(queryItems: [URLQueryItem(name: "q", value: query),
                                URLQueryItem(name: "locale", value: Locale.preferredLanguages[0].components(separatedBy: "-")[0])])
     }
     
     /// Return a Page object from the data
-    func handleResult(with data: Data) -> Page? {
+    public func handleResult(with data: Data) -> Page? {
```

Comment:
handleResult shouldn't have to be public either, it's never called directly from the SDK